test/unittest/jobs/test_task_workflow.py

```diff
@@ -55,7 +50,7 @@ def test_downstream_task_workflows_are_triggered(db, data, flatten):
 
     downstream_listener = downstream_model.to_listener(
         key=upstream_listener.outputs_key,
-        select=upstream_listener.outputs_select,
+        select=db[upstream_listener.outputs].select(),
```

Comment:
I wanted to discuss this

Reply:
I revertd to `upstream_listener.outputs_select`